client/src/app/app-routing.module.ts

```diff
@@ -19,6 +20,10 @@ const routes: Routes = [
     path: 'profile',
     component: ProfileComponent,
     // canActivate: [AuthGuard]
+  },
+  {
+    path: 'keith',
```

Comment:
Same idea as above, lets name this something close to what it should be when you are comfortable.